Dockerfile

```diff
@@ -1,17 +1,17 @@
-#FROM golang:1.21 AS build
```

Comment:
actually we can't do this - this dockerfile is used by go releaser and works as it is 

 what we can do is that have two docker file - Dockerfile and Dockerfile.goreleaser and have the goreleaser.yml point to `Dockerfile.goreleaser`

Reply:
something like in this branch - https://github.com/Layr-Labs/cerberus/tree/madhur/docker-updates - i was planning to add that.